lib/azurex/blob.ex

```diff
@@ -29,11 +30,25 @@ defmodule Azurex.Blob do
   @doc """
   Upload a blob.
 
+  ## The `blob` Argument
+
+  The blob argument may be either a `binary` or a tuple of
+  `{:stream, Stream.t()}`.
+
+  ## The `content_type` Argument
+
+  This argument can be either a valid string, or `nil`. A `content_type`
+  argument of `nil` will result in the blob being assigned the default content
+  type `"application/octet-stream"`.
+
   ## Examples
 
       iex> put_blob("filename.txt", "file contents", "text/plain")
       :ok
 
+      iex> put_blob("filename.txt", {:stream, bitstream}, nil)
```

Comment:
Could you define an example bitstream, so it' s a bit easier to understand the example, something like this:
```elixir
iex> bitstream = <something> 
...> put_blob("filename.txt", {:stream, bitstream}, nil)
:ok
```
where you replace `<something>` with an example